src/Features/Core/Portable/CodeFixes/AddImport/AbstractAddImportCodeFixProvider.SymbolResult.cs

```diff
@@ -38,12 +39,19 @@ public SearchResult(string desiredName, TSimpleNameSyntax nameNode, IReadOnlyLis
                 NameParts = nameParts;
             }
 
-            public bool ShouldRenameNode()
+            public bool DesiredNameDiffersFromSourceName()
             {
                 return !string.IsNullOrEmpty(this.DesiredName) &&
                     this.NameNode != null &&
                     this.NameNode.GetFirstToken().ValueText != this.DesiredName;
             }
+
+            public bool DesiredNameDiffersFromSourceNameOnlyByCase()
+            {
+                Debug.Assert(DesiredNameDiffersFromSourceName());
+                return StringComparer.OrdinalIgnoreCase.Equals(
```

Comment:
There's a `CaseInsensitiveComparison` public API that implements the actual logic the compiler uses for identifier equality.